### ISSUE TITLE
playground subcommand doesn't load

### ISSUE THREAD

**Issue by afroozeh** [tree-sitter/tree-sitter#4038]
### Problem

https://tree-sitter.github.io/tree-sitter/playground seems broken. See this following screenshot:

<img width="858" alt="Screenshot 2024-12-25 at 19 40 22" src="https://github.com/user-attachments/assets/15dcdb8f-003e-4716-b663-39244d4a7645" />

### Steps to reproduce

Go to https://tree-sitter.github.io/tree-sitter/playground

### Expected behavior

Load the playground

### Tree-sitter version (tree-sitter --version)

NA

### Operating system/version

macOS 15.2

**Comment by WillLillis** [comment]
The docs were recently reworked in #4032, the playground can now be found at https://tree-sitter.github.io/tree-sitter/7-playground.html. This is also accessible through the site's side navigation pane:
![image](https://github.com/user-attachments/assets/cd61e928-bda8-49a1-9ff2-11c091b7d470)

**Comment by WillLillis** [comment]
I don't have a real fix yet, but the issues appears to be with the changes to [playground.js](https://github.com/tree-sitter/tree-sitter/blob/master/docs/assets/js/playground.js) from #4032. As a temporary workaround, replacing the playground.js file with [an older copy](https://github.com/tree-sitter/tree-sitter/blob/201b41cf11fb217a1f1ce03ea25b83e62b7b48cb/docs/assets/js/playground.js) seems to fix things for me locally:

![image](https://github.com/user-attachments/assets/65966f70-fe8a-40bd-a549-19ec4d24b1f9)

**Comment by amaanq** [comment]
> You need to also fix that.

*You* need to mind the attitude, bugs are bad but being demanding isn't going to help anyone fix issues. *You* are running the CLI built from master expecting it to be totally bug free, and not the latest stable release. I do appreciate the bug report, and I'm sorry that the local playground broke on *master*, but I'd also appreciate a more respectful tone when filing issues.

**Comment by amaanq** [comment]
Ah right, the favicon doesn't load. Sorry about that. But FTR, "you need to also fix that" does sound a bit demanding. I do apologize for my incorrect assumption about you running the latest cli, & about the broken favicon, I'll have a fix out for this shortly.

**Comment by amaanq** [comment]
Should be fixed, sorry again :sweat_smile:

**Comment by amaanq** [comment]
The **main** playground link works for me and pretty much everyone else I know, so I don't think that's broken. However, you're right that the **local** playground is still broken *when* the CLI is installed without having built the wasm files locally (yeah, this isn't great), and so it fetches these files, including the playground JS, from the website. Coupling the remote and local playground is sort of problematic, as updating the remote one can break the local one, but it's not a choice I made. The issue is, the playground and docs have been changed quite a lot, so I've decided to release 0.24.6 with the updated local playground file, though 0.25 should be coming *soon* anyway. Sorry about that again
